src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ConfigurationGeneral.xaml

```diff
@@ -58,6 +62,18 @@
   <StringProperty Name="DefaultPlatform"
                   Visible="False" />
 
+  <StringProperty Name="DesignerFunctionVisibility"
```

Comment:
These should match the CPS definition: https://devdiv.visualstudio.com/DevDiv/_git/CPS?path=%2Fsrc%2Fshared%2FRules%2Fgeneral.xaml&version=GBmain&line=101&lineEnd=102&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

Reply:
Thanks, updated.